src/library/itunes/itunesmacosimporter.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <QSqlDatabase>
+#include <QString>
+
+#include "library/itunes/itunesimporter.h"
+#include "library/libraryfeature.h"
+
+class ITunesMacOSImporter : public ITunesImporter {
```

Comment:
How about adopting the official naming by using `AppleMusicApiImporter` instead of inventing new names? Would be much easier to recognize.

https://developer.apple.com/documentation/applemusicapi/